README.md

```diff
@@ -9,28 +8,41 @@ A web-app for conducting a quiz, including a page for players to enter their ans
 
 ![screenshot](screenshot.png "Screenshot")
 
-## Installation from release
+## Installation from prebuilt release
+
+### Manual installation
 
 - Download the [latest release](https://github.com/nymanjens/quizmaster/releases)
 - Unpack the archive and open a terminal in the unpacked folder
 - Run `bin/server` (UNIX) or `bin/server.bat` (Windows)
 - Browse to http://localhost:9000
 
-## Installation with Docker
+### Using prebuilt docker image
 
 - Run the following command:
 
 ```
 docker run nymanjens/quizmaster:latest bin/server
 ```
 
-- Browse to `http://<ip_address>:9000`
+## Building and running your own release with Docker
+
+- Run the following command:
```

Comment:
Prefix this with git checkout github/... ?

Reply:
yes checking out the project is a requirement here